website/docs/reference/resource-configs/pre-hook-post-hook.md

```diff
@@ -112,6 +116,52 @@ Pre- and post-hooks can also call macros that return SQL statements.
 
 
 ## Examples
+
+<VersionBlock firstVersion="1.2">
+
+In previous versions of dbt, the most common use of `post-hook` was to execute `grant` statements, to apply database permissions to models right after creating them. Starting in v1.2, we encourage you to use the [`grants` config](resource-configs/grants) instead.
```

Comment:
Should this be a reusable that we add in other places? Maybe the migration guide?

Reply:
good call! snippetize me cap'n